src/config.py

```diff
@@ -1,14 +1,34 @@
 import os, json, gi, argparse, shutil
 from gi.repository import GLib, Gio
-from localization import _, CACHE, DATA, home, system_dir, flatpak, snap, settings
-
-# add command-line arguments
+from localization import _, CACHE, DATA, system_dir, flatpak, snap, settings
+home = os.getenv('HOME')
```

Comment:
The `home` variable is defined in the` src/localization.py` file, so it is not necessary to use a new, same variable in this file. This is because Snap requires a custom home directory definition.

Reply:
i will remove that line thanks for the feedback on that one i did not know the localization.py did that

Reply:
done